src/main/java/org/opensearch/security/OpenSearchSecurityPlugin.java

```diff
@@ -1151,9 +1152,19 @@ public Settings additionalSettings() {
             builder.put(NetworkModule.TRANSPORT_TYPE_KEY, "org.opensearch.security.ssl.http.netty.SecuritySSLNettyTransport");
             builder.put(NetworkModule.HTTP_TYPE_KEY, "org.opensearch.security.http.SecurityHttpServerTransport");
         }
+
+        builder.put(getCustomSerializationNodeAttribute(settings));
+
         return builder.build();
     }
 
+    private Settings getCustomSerializationNodeAttribute(Settings settings) {
+        return Settings.builder().put(
+                Node.NODE_ATTRIBUTES.getKey() + ConfigConstants.CUSTOM_SERIALIZATION_ENABLED,
+                SecuritySettings.CUSTOM_SERIALIZATION_ENABLED.get(settings)
+        ).build();
+    }
+
```

Comment:
Should we do a patch release instead for this to avoid identification through node attributes?

Reply:
Done